.env.example

```
@@ -204,6 +204,8 @@ WEBHOOK_DEV_SC_AUDIT=
 WEBHOOK_DEV_SC_AUDIT_BURRASEC=
 # proposal-created / please-sign notifications from multiNetworkExecution.sh
 WEBHOOK_DEV_SC_MULTISIG_PROPOSALS=
+# scheduled-job output (deferred-cleanup reconcile TTL alert); only delivered when CI is set
+WEBHOOK_DEV_SC_GITHUB_CI_NOTIFICATIONS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WEBHOOK_DEV_SC_GITHUB_CI_NOTIFICATIONS key should go before the WEBHOOK_DEV_SC_MULTISIG_PROPOSALS key

(UnorderedKey)